### PR TITLE
Add profile for twitch,youtube,youtube-music; fix git-cola ,add cola

### PR DESCRIPTION
**Pull request by kortewegdevries**

### CODE REVIEW

File: etc/profile-a-l/git-cola.profile [hunk 1]
```diff
@@ -62,5 +62,5 @@ dbus-user filter
 dbus-system none
 
 read-only ${HOME}/.ssh
-read-only ${HOME}/.gnupg
+# read-only ${HOME}/.gnupg
```
Comment (reinerh) [inline etc/profile-a-l/git-cola.profile:65]: why was this necessary?

Comment (kortewegdevries) [reply etc/profile-a-l/git-cola.profile:65]: `gpg` creates a temporary file under `.gnupg` https://imgur.com/N1n5Jpt, I forgot when I made the profile and cola passes this to git: git -c diff.suppressBlankEmpty=false -c log.showSignature=false commit -F git-cola-commit-message-lk887bkx -v --gpg-sign`. I can uncomment and put a comment above like the `private-etc` one.

**Comment by kortewegdevries** [comment]
There's currently a `ssh-askpass`(it's in /usr/share/git-cola/ssh-askpas) failing on a newer git-cola version on Arch how exactly do I fix it? whitelist all git,gitk,git-gui,git-cola related directories in `/usr/share/` or get rid of the current whitelists or just add a comment if people wants to use ssh in git?

**Comment by rusty-snake** [comment]
> how exactly do I fix it?

how exactly is it failing?

> whitelist all git,gitk,git-gui,git-cola related directories in /usr/share/

For a git clinet? I'm fine with that.

**Comment by kortewegdevries** [comment]
> how exactly is it failing?

GitHub is already in my list of know-hosts, I'll comment the read-only similarly but it also needs access to that.

https://imgur.com/ZgfB0iM

If it's okay, I'll add them some time later, I have some work...

**Comment by kortewegdevries** [comment]
I forgot to sync this branch with upstream master...is the merge conflict gone or do I need to do something else?

File: etc/inc/disable-programs.inc [hunk 2]
```diff
@@ -396,6 +398,8 @@ blacklist ${HOME}/.config/yandex-browser
 blacklist ${HOME}/.config/yandex-browser-beta
 blacklist ${HOME}/.config/yelp
 blacklist ${HOME}/.config/youtube-dl
+blacklist ${HOME}/.config/youtubemusic-nativefier-040164
```
Comment (rusty-snake) [inline etc/inc/disable-programs.inc:401]: The "040164" look generated. Is it stable?

File: etc/profile-a-l/git-cola.profile [hunk 3]
```diff
@@ -49,18 +62,20 @@ seccomp
 shell none
 tracelog
 
-# private-bin atom,bash,colordiff,emacs,fldiff,geany,gedit,git,git gui,git-cola,git-dag,gitk,gpg,gvim,leafpad,meld,mousepad,nano,notepadqq,python*,sh,ssh,vim,vimdiff,which,xed
+private-bin atom,basename,bash,colordiff,diff,emacs,envsubst,fldiff,geany,gedit,gettext,git,git gui,git-cola,git-dag,gitk,gpg,gpg-agent,gvim,leafpad,meld,mousepad,nano,notepadqq,nvim,pinentry,pinentry-gtk-2,ps,python*,sh,ssh,ssh-agent,tclsh,tr,vim,vimdiff,wc,which,xed
```
Comment (rusty-snake) [inline etc/profile-a-l/git-cola.profile:65]: ",git gui," should the be "git-gui"?

There are also other pinentry binaries.

All the editors blow this up. Proposal:

```
private-bin basename,bash,colordiff,diff,…
# Replace with your editor.
private-bin atom,emacs,geany,gedit,…
```

File: etc/profile-a-l/git-cola.profile [hunk 4]
```diff
@@ -49,18 +62,20 @@ seccomp
...
+# Breaks meld as diff viewer
+# dbus-user filter
 # Uncomment if you need keyring access
 # dbus-user.talk org.freedesktop.secrets
 dbus-system none
 
-read-only ${HOME}/.ssh
-read-only ${HOME}/.gnupg
 read-only ${HOME}/.git-credentials
+# Comment if you sign commits with GPG
```
Comment (rusty-snake) [inline etc/profile-a-l/git-cola.profile:78]:
```suggestion
# Uncomment if you sign commits with GPG
```

File: etc/profile-m-z/twitch.profile [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/Twitch
+whitelist ${HOME}/.config/Twitch
+include whitelist-common.inc 
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+nou2f
```
Comment (rusty-snake) [inline etc/profile-m-z/twitch.profile:24]: Has twitch 2fa?

File: etc/profile-m-z/twitch.profile [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+novideo
```
Comment (rusty-snake) [inline etc/profile-m-z/twitch.profile:25]: Has twitch a webcam function?

File: etc/profile-m-z/youtube.profile [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/Youtube
+whitelist ${HOME}/.config/Youtube
+include whitelist-common.inc 
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+nou2f
```
Comment (rusty-snake) [inline etc/profile-m-z/youtube.profile:24]: YT has 2FA AFAIK, this wrapper too?

Comment (kortewegdevries) [reply etc/profile-m-z/youtube.profile:24]: It's a wrapper, but it should work? I forgot...I don't use it.

File: etc/profile-m-z/twitch.profile (hunk 6 above)
Comment (kortewegdevries) [reply etc/profile-m-z/twitch.profile:25]: Thanks.

File: etc/profile-m-z/twitch.profile (hunk 5 above)
Comment (kortewegdevries) [reply etc/profile-m-z/twitch.profile:24]: It's a wrapper, but it should work?

File: etc/profile-a-l/git-cola.profile (hunk 3 above)
Comment (kortewegdevries) [reply etc/profile-a-l/git-cola.profile:65]: Is opposite for me ? https://imgur.com/SUmk6Ve, I'll change the editor and pinentry ones, I had only those two+-emacs,-curses...

File: etc/inc/disable-programs.inc (hunk 2 above)
Comment (kortewegdevries) [reply etc/inc/disable-programs.inc:401]: I need to check I uninstalled the thing.

Comment (kortewegdevries) [reply etc/inc/disable-programs.inc:401]: Reinstalled still `youtubemusic-nativefier-040164`

File: etc/profile-a-l/git-cola.profile (hunk 3 above)
Comment (rusty-snake) [reply etc/profile-a-l/git-cola.profile:65]: 1. Then is the binary still `git` and not `git gui` (or do you have a `/usr/bin/git gui`).
2. I have
```
/usr/bin/pinentry
/usr/bin/pinentry-curses
/usr/bin/pinentry-gnome3
/usr/bin/pinentry-gtk -> pinentry-gtk-2
/usr/bin/pinentry-gtk-2
```

Comment (kortewegdevries) [reply etc/profile-a-l/git-cola.profile:65]: 1.> I have neither in `/bin/`?
2.> Yeah same for me, I choose gtk-2 to cache...

**Comment by kortewegdevries** [comment]
Is this okay or something is needed?

**Comment by reinerh** [comment]
Looks good to me, but please wait for @rusty-snake 's reply. :-)

**Comment by rusty-snake** [comment]
Everythings fine, expect that I'm unsure if we want GPG/SSH for git-cola by default. We have it enabled in all the others git-clients (git,meld,gitter,github-desktop,gitg).

**Comment by kortewegdevries** [comment]
> expect that I'm unsure if we want GPG/SSH for git-cola by default

Since emails are getting I follow with git too. 

gitg needs fixes too, you can't push changes if remote is over ssh: see error cannot connect to SSH-agent in the menu bar below.

File: etc/profile-a-l/cola.profile [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# Firejail profile for cola
+# Description: Linux native frontend for Git,alternative call for git-cola
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cola.local
+# Persistent global definitions
+include globals.local
+
+# Redirect
+
```
Comment (rusty-snake) [inline etc/profile-a-l/cola.profile:10]:
```suggestion
```

**Review by rusty-snake — CHANGES_REQUESTED** [review]
Use sort.py.

**Comment by kortewegdevries** [comment]
> Use sort.py.

Ran it through 3607,3603,3577.

**Review by rusty-snake — APPROVED** [review]

**Comment by rusty-snake** [comment]
Thanks for this contribution!
